pkg/controllers/management/auth/manager.go

```diff
@@ -985,3 +1006,32 @@ func getLabelRequirements(objMeta metav1.ObjectMeta) ([]labels.Requirement, erro
 	}
 	return []labels.Requirement{*reqUpdatedLabel, *reqNsAndNameLabel}, nil
 }
+
+// canUpdatePSA returns true if the roletemplate has the updatepsa verb enabled
+func (m *manager) canUpdatePSA(roleTemplate string) (bool, error) {
+	var additionalVerbs bool
```

Comment:
nit: why is this var called `additionalVerbs`? Would make more sense to name it something like `canUpdatePSA`?